.github/workflows/proc_self_stat

```diff
@@ -0,0 +1 @@
+/proc/self/stat
```

Comment:
This is some of your best work right here

Reply:
This file is giving me trouble on Windows. When I check out the branch, the file shows up as modified in my working set. WIndows doesn't support symlinks like this. I"m not sure what to do.

Reply:
That means a large percentage of users will have the same problem, leading to questions even if I am sure there is some symlink setting in windows git somewhere which might fix this.

Well, I guess that means I will drop that nice hack.

Reply:
ok, pushed a new version with the hack dropped.